common-lib/src/cmr/common/api/context.clj

```diff
@@ -48,3 +51,12 @@
   (when-let [request-id (context->request-id context)]
     {REQUEST_ID_HEADER request-id}))
 
+(defn context->user-id
```

Comment:
Cool  :-) Yeah, wherever folks think this should live, that's where I'll put it; just say the word.